src/main/java/stringcalculator/domain/Calculator.java

```diff
@@ -0,0 +1,36 @@
+package stringcalculator.domain;
+
+import java.util.Objects;
+
+public class Calculator {
+
+    private final SimpleParser parser;
```

Comment:
인터페이스를 만든 의미를 되살려 보면 어떨까요?

```suggestion
    private final Parser parser;
```